engine/lib/metadata.php

```diff
@@ -58,16 +56,16 @@ function elgg_delete_metadata_by_id($id) {
  * @param string $name           Name of the metadata
  * @param string $value          Value of the metadata
  * @param string $value_type     'text', 'integer', or '' for automatic detection
- * @param int    $owner_guid     GUID of entity that owns the metadata. Default is logged in user.
- * @param null   $ignored        This argument is not used
+ * @param int    $ignored1       This argument is not used
+ * @param null   $ignored2       This argument is not used
  * @param bool   $allow_multiple Allow multiple values for one key. Default is false
  *
  * @return int|false id of metadata or false if failure
  */
-function create_metadata($entity_guid, $name, $value, $value_type = '', $owner_guid = 0,
-		$ignored = null, $allow_multiple = false) {
+function create_metadata($entity_guid, $name, $value, $value_type = '', $ignored1 = null,
```

Comment:
This function no longer makes sense. Better deprecate and use entity setter

Reply:
good points... did not think about that... what to do with `$multiple`?

Reply:
Well multiple is same as setting value to an array, no?

Reply:
going to deprecate this in a separate PR